chat-android/src/main/java/com/ably/chat/ChatApi.kt

```diff
@@ -158,17 +160,17 @@ internal class ChatApi(
 
     private suspend fun makeAuthorizedRequest(
         url: String,
-        method: String,
+        method: HttpMethod,
         body: JsonElement? = null,
     ): JsonElement? = suspendCancellableCoroutine { continuation ->
         val requestBody = body.toRequestBody()
         realtimeClient.requestAsync(
-            method,
-            url,
-            arrayOf(apiProtocolParam),
-            requestBody,
-            arrayOf(),
-            object : AsyncHttpPaginatedResponse.Callback {
+            path = url,
+            method = method,
+            headers = listOf(apiProtocolParam),
```

Comment:
Here, `apiProtocolParam` param is passed as a part of headers, and in `makeAuthorizedPaginatedRequest` it's passed as a params, is this accepted 🤨? 
Even if it is, I think we should be consistent about it's usage. I think as per it's name `apiProtocolParam` should be passed as params

Reply:
it's a bug, nice spot!